### PR TITLE
chore(cli, core, schema): Release 12.0.0

### PR DESCRIPTION
**Pull request by xavdid**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,117 +1,138 @@
+## 12.0.0
+
+_Released `2022-03-23`_
+
+We're breaking slightly from our pattern of a single yearly major release. The `12.0.0` release contains some backwards-incompatible changes to how middleware and auth refreshes work. For the most part, you'll be able to upgrade to this version safely, but as always, it's worth re-running unit tests (especially those related to authentication).
```
Comment (xavdid) [inline CHANGELOG.md:5]: Tried to add a little context here and let them know we're doing another major version soonish.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,117 +1,138 @@
...
 
 ### core
 
-* :bug: Fix another hanging issue where an action could be invoked multiple times ([#513](https://github.com/zapier/zapier-platform/pull/513))
+- :bug: Fix another hanging issue where an action could be invoked multiple times ([#513](https://github.com/zapier/zapier-platform/pull/513))
 
 ### schema
 
-* None!
+- None!
```
Comment (xavdid) [inline CHANGELOG.md:36]: prettier converted all the bullets from `*` into `-`. I'm not sure if you run prettier on markdown, but I figured it's nice to be consistent here (and my `prettier` gets run on-save)

**Review by eliangcs — APPROVED** [review]
Looks good!

**Comment by rafaeelaudibert** [comment]
Hey :)
As funny as it might sound, I'm probably one of those developers migrating from v9.0 to v11.0 that would care, and I'd really fancy this option to globally disable `skipThrowForStatus`.

I couldn't find any public information about when this is going to be released. Do you happen to know when this is going to be merged?

Cheers!

**Comment by xavdid** [comment]
hey @rafaeelaudibert! I'm planning on cutting a release shortly, in the next hour or so. It'll be available for use right after that!

**Pull request merged**